app/views/modals/logout.html

```diff
@@ -0,0 +1,13 @@
+<div class="modal-resource-action">
+  <!-- Use a form so that the enter key submits when typing a project name to confirm. -->
+  <form>
+    <div class="modal-body">
+      <h1>You have been inactive for period of time.</h1>
+      <h2>You are going to be logged out in <strong>{{timeToLogout}}</strong></h2>
+    </div>
+    <div class="modal-footer">
+      <button class="btn btn-lg btn-danger" type="submit" ng-click="logout();">Log Out</button>
```

Comment:
I'm not sure what the appropriate order if for these buttons.  @ncameronbritt can you look into what the typical behavior is for these automated logout dialogs.  I don't want someone to panic and click logout when they meant to cancel the logout.

Reply:
Not sure you need two buttons. The action should be "stay logged in".  The user will be logged out anyway if she does nothing, hence this dialog. So is there any compelling reason to offer a "log out" action? I think it could be as simple as something like this: 
<img width="563" alt="screen shot 2017-11-22 at 8 54 47 am" src="https://user-images.githubusercontent.com/8974593/33130885-c68b941c-cf62-11e7-9a96-092606850ec9.png">

Another alternative is that you don't need any buttons. I like what they do [here](https://www.cloudera.com/documentation/enterprise/5-7-x/topics/cm_auto_logout.html) with a kind of "press any key to continue" prompt. If you're being logged out due to inactivity, it makes sense that any activity should keep the user logged in. 

Another thing I like about the example in that link is the way the log in page tells the user why they're seeing it again: "You were logged out due to inactivity."

cc @openshift/team-ux-review